.pre-commit-config.yaml

```diff
@@ -21,3 +21,10 @@ repos:
     hooks:
       - id: terraform_docs
       - id: terraform_fmt
+
+  - repo: git://github.com/golangci/golangci-lint
+    rev: v1.21.0
+    hooks:
+      - id: golangci-lint
+        entry: golangci-lint run --verbose
```

Comment:
This should probably be updated in the template repo, but you can remove
```
golangci-lint run --verbose
verbose: true
```

Reply:
Removed. Hopefully I captured your intent. Thanks!